inc/alert.class.php

```diff
@@ -416,7 +416,7 @@ public function showForm($ID, $options = []) {
 
    static function displayOnCentral() {
       echo "<tr><th colspan='2'>";
-      self::displayAlerts(['show_only_login_alerts' => false]);
+      self::displayAlerts(['show_only_helpdesk_alerts' => true]);
```

Comment:
This method is used to display alerts on both simplified and central interface.
I did not tested it, but I think the fix should be this:
```suggestion
      self::displayAlerts(['show_only_helpdesk_alerts' => Session::getCurrentInterface() == 'helpdesk']);
```

Reply:
tested and approved